dnf-plugins-core.spec

```diff
@@ -9,7 +10,8 @@
 %endif
 
 Name:           dnf-plugins-core
-Version:        1.0.0
+Epoch:          1
```

Comment:
Epoch is not nice but it's much easier to maintain upgrades if all packages have the same evr rather than some having provides with different epoch. Bumping the epoch in the past was a mistake but now it's too late. Let's not complicate things again.